canonical-websites/www.ubuntu.com
# Updated download path for IBM X

## Done

- Updated download path for IBM X so that it requires you to fill in the form before the download starts.
- Moved the form to the first page and the download starts on the thank-you like most other sections
- Updated z Systems to IBM Z as well BUT I have NOT updated the copy docs as they are setup for 18.04, I will update soon as possible.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - [download linuxone](http://0.0.0.0:8001/download/server/linuxone)
  - [thank you](http://0.0.0.0:8001/download/server/thank-you-linuxone)
    - remember, when testing the form, the redirect will be wrong as it is hard coded in marketo

## Issue / Card

Fixes #2948
